src/Employer/Employer.Web/Controllers/ApplicationReviewController.cs

```diff
@@ -28,15 +31,37 @@ public async Task<IActionResult> ApplicationReview(ApplicationReviewRouteModel r
         [HttpPost("", Name = RouteNames.ApplicationReview_Post)]
         public async Task<IActionResult> ApplicationReview(ApplicationReviewEditModel m)
         {
-            if (ModelState.IsValid == false)
+            if (!ModelState.IsValid)
             {
                 var vm = await _orchestrator.GetApplicationReviewViewModelAsync(m);
                 return View(vm);
-            }
-
-            await _orchestrator.PostApplicationReviewEditModelAsync(m, User.ToVacancyUser());
+            }            
+            var applicationStatusConfirmationViewModel = await _orchestrator.GetApplicationStatusConfirmationViewModelAsync(m);
+            return RedirectToRoute(RouteNames.ApplicationReviewConfirmation_Get, applicationStatusConfirmationViewModel);            
+        }
 
-            return RedirectToRoute(RouteNames.VacancyManage_Get);
+        [HttpGet("status", Name = RouteNames.ApplicationReviewConfirmation_Get)]
+        public IActionResult ApplicationStatusConfirmation(ApplicationStatusConfirmationViewModel applicationStatusConfirmationViewModel)
+        {            
+            return View(applicationStatusConfirmationViewModel);
```

Comment:
I'm concerned about all the data getting passed in via the querystring (the candidate feedback might be quite big?). Maybe we should use TempData?  We should also recheck that they have authorised access to the application.  If TempData is null for whatever reason we could just redirect back to ApplicationReview_Get